ras/ras_ppcutils.py

```diff
@@ -291,10 +291,22 @@ def test_ls_vdev(self):
         """
         self.log.info("===============Executing ls-vdev tool test===="
                       "===========")
+        self.is_fail = 0
         self.run_cmd("ls-vdev")
         self.run_cmd("ls-vdev -h")
         self.run_cmd("ls-vdev -V")
-        self.error_check()
+        dev_name = self.run_cmd_out("ls-vdev").split()[1]
+        flag = True
+        for line in self.run_cmd_out("lsblk").splitlines():
```

Comment:
I think we have an utils part to get the output of `lsblk` command .
Please check on it.
if not there, we need push this code to utils to get the device names from lsblk as most of the ras tools outputs are compared w.r.t lsblk outputs.

Reply:
@Naresh-ibm modified the utils/disk.py program in order to get physical disks and their children sent PR for that 
https://github.com/avocado-framework/avocado/pull/5677
And according to the above PR modifying this script and re-pushing it again..
Please review both and merge them if they looks good